shell/platform/embedder/embedder_external_view_embedder.cc

```diff
@@ -196,6 +202,19 @@ bool EmbedderExternalViewEmbedder::SubmitFrame(GrContext* context) {
       return false;
     }
 
+    // Indicate a layer for the platform view. Add to `presented_platform_views`
+    // in order to keep at allocated just for the scope of the current method.
+    // The layers presented to the embedder will contain a back pointer to this
+    // struct. It is safe to deallocate when the embedder callback is done.
+    presented_platform_views[view_id] = MakePlatformView(view_id);
+    presented_layers.push_back(
+        MakeLayer(params, presented_platform_views.at(view_id)));
+
+    if (!pending_canvas_spies_[view_id]->DidDrawIntoCanvas()) {
+      // Nothing was drawn into the overlay canvas, we don't need to composite
+      // it.
+      continue;
```

Comment:
Test this `continue`. We should upgrade the embedder test compositor to make sure we track the number of backing stores being requested of the the embedder. Then assert that we don't ask for extra backing stores. If we forgot or later moved this `continue`, no test would fail.